components/collector/src/source_collectors/gitlab/job_runs_within_time_period.py

```diff
@@ -0,0 +1,29 @@
+"""GitLab job runs within time period collector."""
+
+from collections.abc import Sequence
+from typing import cast
+
+from dateutil.parser import parse
+
+from collector_utilities.functions import days_ago
+from collector_utilities.type import Job
+from model import SourceResponses
+
+from .base import GitLabJobsBase
+
+
+class GitLabJobRunsWithinTimePeriod(GitLabJobsBase):
+    """Collector class to measure the amount of GitLab CI builds ran within a specified time period."""
```

Comment:
Voltooid deelwoord van 'to run' is 'run':

```suggestion
    """Collector class to measure the amount of GitLab CI builds run within a specified time period."""
```